src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,9 @@
+package com.example;
+
+public class Customer {
+    private String LoyaltyCard;
+
+    public Customer(String LoyaltyCard) {
+        this.LoyaltyCard = LoyaltyCard;
```

Comment:
What you need to have is another class called `LoyaltyCard`. Then, objects of this class would be initialized with a string containing a card number.